go.mod

```diff
@@ -50,7 +50,6 @@ require (
 	github.com/hashicorp/hcl/v2 v2.17.0 // indirect
 	github.com/hashicorp/logutils v1.0.0 // indirect
 	github.com/hashicorp/terraform-json v0.21.0 // indirect
-	github.com/hashicorp/terraform-plugin-docs v0.18.0 // indirect
```

Comment:
Why did you decide to remove this plugin?

Reply:
return back, ide show it isn't used and remove it each time.